README.md

```diff
@@ -19,15 +19,14 @@
 This repository contains two standalone modules that interact with commercetools and Adyen.
 Complete integration requires running both of the modules.
 
-![Overview diagram](https://user-images.githubusercontent.com/3469524/86220256-9f8ab900-bb83-11ea-963a-243e9992283f.jpg)
-1. The shopper does a checkout and starts the payment process.
-2. Shop communicates only with the commercetools to process payment.
-3. The commercetools platform communicates with the extension module.
-4. The extension module communicates with the Adyen payment provider.
-5. Adyen returns a payment result to the extension module.
-6. The extension module updates the commercetools payment.
-7. The shop verifies the updated payment.
-8. The shop presents the results to the shopper.
+![Payment flow](./docs/images/payment-flow.svg)
+1. Front end uses [Adyen Web Coponents](https://docs.adyen.com/checkout/supported-payment-methods) to present required payment methods. The list of available payment methods can be also obtained through the integration with the help of [get available payment methods request](./extension/docs/WebComponentsIntegrationGuide.md#step-3-get-available-payment-methods-optional). On user interaction, frontend web-component generates JSON payloads like [make payment](https://docs.adyen.com/payment-methods/cards/web-component#make-a-payment) or submit additional payment data which has to be provided to commercetools payment as described [here](./extension/docs/WebComponentsIntegrationGuide.md#web-components-integration-guide).  
+2. With help of the API-Extension commercetools sends provided data to the Extension Module.
+3. The Extension Module processes provided data by the front end, exchanges it with Adyen and provides synchronous response back to the front end / commercetools caller. Based on the result the front end either creates an order or continues with further payment steps as described [here](./extension/docs/WebComponentsIntegrationGuide.md#web-components-integration-guide).
```

Comment:
```suggestion
3. The Extension Module processes provided web component payload passed by the front end, exchanges it with Adyen API, and provides synchronous response back to the front end / commercetools caller. Based on result, the front end either creates an order or continues with further payment steps as described at [integration guide](./extension/docs/WebComponentsIntegrationGuide.md#web-components-integration-guide).
```
> provides synchronous response back to the front end / commercetools caller.

I am not sure about this part. Is the extension module response back to the front end?

> the front end either creates an order

Does the front end create an order? => I guess for simplicity you are considering the merchant server as the front end ?

Reply:
> I am not sure about this part. Is the extension module response back to the front end?

I try to keep it as simple as possible and follow what one sees on the diagram - without not going to much into details.

> Does the front end create an order? => I guess for simplicity you are considering the merchant server as the front end ?

For simplicity there is no merchant server on the diagram but front end is :)